mailpoet/lib/Segments/DynamicSegments/Filters/SubscriberSubscribedViaForm.php

```diff
@@ -0,0 +1,55 @@
+<?php // phpcs:ignore SlevomatCodingStandard.TypeHints.DeclareStrictTypes.DeclareStrictTypesMissing
```

Comment:
why can't you use strict_types?

Reply:
This was actually just copied from the filter I copied as a starting point, and I don't think it's actually necessary. I'm going to move this forward to QA for now just so there isn't a delay, I've added this to my list of things to clean up/refactor, which I'm tracking in MAILPOET-5227